metals/src/main/scala/scala/meta/internal/metals/codeactions/ImportMissingSymbol.scala

```diff
@@ -43,14 +43,42 @@ class ImportMissingSymbol(compilers: Compilers) extends CodeAction {
         }
     }
 
+    def importMissingSymbols(
+        codeActions: Seq[l.CodeAction]
+    ): Seq[l.CodeAction] = {
+      val uniqueCodeActions = codeActions
+        .groupBy(_.getDiagnostics())
+        .values
+        .filter(_.length == 1)
+        .flatten
+        .toSeq
+
+      val allCodeActions = if (uniqueCodeActions.length > 1) {
+        val allSymbols: l.CodeAction = new l.CodeAction()
+
+        val uri = params.getTextDocument().getUri()
+        val diags = uniqueCodeActions.flatMap(_.getDiagnostics().asScala)
+        val edits = uniqueCodeActions
+          .flatMap(_.getEdit().getChanges().get(uri).asScala)
+          .distinct
+
+        allSymbols.setTitle(ImportMissingSymbol.allSymbolsTitle)
+        allSymbols.setKind(l.CodeActionKind.QuickFix)
+        allSymbols.setDiagnostics(diags.asJava)
+        allSymbols.setEdit(new l.WorkspaceEdit(Map(uri -> edits.asJava).asJava))
```

Comment:
I think we could just join all the textEdits into a single text edit, which might help with the tests and the additional newline.

Reply:
I managed to fix everything that way without changing anything to the TextEdits.scala file. I don't know if this is the right way but it seems that imports are always coming in different order so maybe it is.